build/Makefile

```diff
@@ -70,7 +70,7 @@ GS_TEST_IMAGE ?= us-docker.pkg.dev/agones-images/examples/simple-game-server:0.3
 BETA_FEATURE_GATES ?= "CountsAndLists=true&DisableResyncOnSDKServer=true"
 
 # Enable all alpha feature gates. Keep in sync with `false` (alpha) entries in pkg/util/runtime/features.go:featureDefaults
-ALPHA_FEATURE_GATES ?= "PlayerAllocationFilter=true&PlayerTracking=true&RollingUpdateFix=true&PortRanges=true&PortPolicyNone=true&Example=true"
+ALPHA_FEATURE_GATES ?= "PlayerAllocationFilter=true&PlayerTracking=true&RollingUpdateFix=true&PortRanges=true&PortPolicyNone=true&AutopilotPassthroughPort=false&Example=true"
```

Comment:
The instructions might not be totally clear but `ALPHA_FEATURE_GATES` is the inverse of the defaults. So here you way `AutopilotPassthroughPort=true`

Reply:
Ohhhh okok i got confused by this line `Keep in sync with `false` (alpha) entries in pkg/util/runtime/features.go:featureDefaults`, will change!